### PR TITLE
Solution attempt

### PR DESCRIPTION
**Pull request by Likh-Alex**

### CODE REVIEW

File: src/main/java/core/basesyntax/PasswordValidator.java [hunk 1]
```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/PasswordValidator.java:6]: remove empty line

File: src/main/java/core/basesyntax/PasswordValidator.java [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+        if (password != null && (password.equals(repeatPassword)
+                && password.length() >= 10)) {
+            return true;
+        }
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/PasswordValidator.java:11]: let's remove empty line)

File: src/main/java/core/basesyntax/UserService.java [hunk 3]
```diff
@@ -3,9 +3,18 @@
 public class UserService {
     public void registerUser(User user) {
 
+        PasswordValidator passwordValidator = new PasswordValidator();
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/UserService.java:7]: remove all empty lines in the method
```suggestion
        PasswordValidator passwordValidator = new PasswordValidator();
```

**Review by Sofasmile — CHANGES_REQUESTED** [review]
good job

File: src/main/java/core/basesyntax/PasswordValidator.java (hunk 1 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/PasswordValidator.java]: Removed, thank you for the note)

File: src/main/java/core/basesyntax/PasswordValidator.java (hunk 2 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/PasswordValidator.java]: This one also removed)

File: src/main/java/core/basesyntax/UserService.java (hunk 3 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/UserService.java:7]: All empty lines in this method are removed)

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**